.github/workflows/changelog_verifier.yml

```
@@ -5,6 +5,7 @@ on:
       - 'whitesource-remediate/**'
       - 'backport/**'
   pull_request:
+    branches: main
```

```suggestion
    branches:
      - main
```
I think this is the correct syntax.

---

That's how we have it [here](https://github.com/opensearch-project/flow-framework/blob/main/.github/workflows/add-backport-label.yml#L4) as well.